server/src/main/java/io/spine/server/commandbus/CommandAckMonitor.java

```diff
@@ -50,10 +57,14 @@
 final class CommandAckMonitor extends DelegatingObserver<Ack> {
 
     private final SystemWriteSide writeSide;
+    private final ImmutableMap<CommandId, Command> commands;
 
     private CommandAckMonitor(Builder builder) {
         super(builder.delegate);
         this.writeSide = delegatingTo(builder.systemWriteSide).get(builder.tenantId);
+        this.commands = builder.commands
+                .stream()
```

Comment:
Please fix the alignment.

Reply:
BTW, this is how automatic formatting works (since `commands` is a field). I'll take a look if we could fix it via the IDEA config later.